src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -231,7 +231,7 @@ const batchRequestTxsData = (txs: any[]) => {
   const web3Batch = new web3.BatchRequest()
 
   const whenTxsValues = txs.map((tx) => {
```

Comment:
I understand this is not related to your PR, but could we please come up with a better name for this array? I had to look at the code to understand what it's doing. The name currently makes no sense

Reply:
yup, I couldn't find a better name, what about now https://github.com/gnosis/safe-react/pull/812/commits/e4620901c02c25fc262844273bf6c6f883488a97

Reply:
this one is way better, thanks :)